README.md

```diff
@@ -118,7 +119,9 @@ end
 ```
 
 ```rb
-# Your answer goes here...
+<div ng-repeat="post in vm.posts">
```

Comment:
Looking for Ruby/Rails code, not Javascript. How would you modify the Rails app so that it serves up JSON?